Vagrantfile

```diff
@@ -50,14 +50,8 @@ Vagrant.configure("2") do |config|
   config.ssh.forward_x11 = true
 
   # setup shared folder
-  config.vm.synced_folder ".", "/vagrant", type: "rsync", rsync__exclude: [
-    "build",
-    "src/rhsm/_certificate.so",
-    "subscription-manager.egg-info",
-    "cockpit/node_modules",
-    "updates.img",
-    "vagrant/vagrant_data",
-  ], rsync__args: ["--verbose", "--archive", "--delete", "-z", "--copy-links", "-W"]
+  config.vm.synced_folder ".", "/vagrant", type: "sshfs",
+                          sshfs_opts_append: "-o nonempty"
```

Comment:
It seems that it is not possible to exclude some directories from syncing (e.g. build) :-(

When tito is used, then we can use following approach:
```bash
rm -rf /home/vagrant/tito
mkdir /home/vagrant/tito
tito build --test --rpm --verbose -o /home/vagrant/tito --no-cleanup
```

Reply:
Yeah and tested that:
```
config.vm.synced_folder "./build", "/vagrant/build", type: "sshfs",
                          sshfs_opts_append: "-o nonempty", disabled: true
```
has no effect, because parent directory of `./build` is synced.